nodejs-http-server-template/wrangler.jsonc

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "node_modules/wrangler/config-schema.json",
+  "name": "nodejs-http-server-template",
+  "main": "src/index.ts",
+  "compatibility_date": "2025-09-03",
+  "compatibility_flags": ["nodejs_compat", "enable_nodejs_http_server_modules"],
```

On by default since 2025-08-15, right?

```suggestion
  "compatibility_flags": ["nodejs_compat"],
```

---

This doesn't work. @vicb can you help me why this is failing? Only nodejs_compat results in this error

```
✘ [ERROR] service core:user:nodejs-http-server-template: Uncaught Error: [unenv] http.createServer is not implemented yet!

    at null.<anonymous> (index.js:7:10) in createNotImplementedError
    at null.<anonymous> (index.js:13:27) in fn
    at null.<anonymous> (index.js:1079:14)
    ```

---

Nice catch. Thank you Greg!